blocked-edges/4.18.0-ec.3-CMOSuddenStrictConfigValidation.yaml

```diff
@@ -0,0 +1,16 @@
+to: 4.18.0-ec.3
+from: .*
```

Comment:
should we enumerate them all?

Reply:
Yeah, this regex should cover the versions from which the update is risky. See e.g. blocked-edges/4.16.9-HyperShiftNodePoolSkewBinaryDownload.yaml for an example. Fortunately we're quite fine here I think - we'll need 4.17.[0-4] + maybe `4.18.0-ec.0`?